src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__info {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    margin-right: 68px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &_imagep {
```

Comment:
The BEM naming convention should be consistent. Consider renaming `&_imagep` to `&__image` to follow the BEM standard.